### ISSUE TITLE
Hinges often break and wander off. (When docking and undocking)

### ISSUE THREAD

**Issue by EitanTal** [sirkut/InfernalRobotics#16]
This is an issue I encounter very often, especially when combining a clamp-o-tron with a hinge. The root cause is a geometric issue 

I have a rover that grabs a tool with a clamp. The tool consists of two hinges and a clamp at the end (Macaroni-L shaped).

The tool works great in the beginning, but gets damaged after moving things around. (docking/undocking), causing unwanted moves and stresses. The problem gets as severe as tipping over a 20-ton construction vehicle and making it fly in the air like a crazy poltergeist!

Broken hinge:.

![2014-06-04_00007](https://cloud.githubusercontent.com/assets/7789714/3172775/96b9aec8-ebda-11e3-8c57-6aa221591fab.jpg)

Same broken hinge before I tried to move it:

![2014-06-04_00005](https://cloud.githubusercontent.com/assets/7789714/3172782/b784f892-ebda-11e3-861a-49e1d3c0c095.jpg)

I can supply a Craft file that makes the problem appear.

Also, another interesting side-effect is that the gantries get affected too, somehow, with their bases now hovering in the air.

**Comment by sirkut** [comment]
This is a known issue although I specified specifically not to attach docking ports to pistons. This also applies to hinges as it will cause strange things to happen. I have a vague idea that I need to redo the parenting portion of the code when docked. Thanks for submitting the issue, until then tread carefully when using IR parts directly connected to docking ports.

**Comment by EitanTal** [comment]
Right, ok. That should be a broader warning, as I didn't use a piston, but a gantry, and they were two different objects that joined together, and the gantries were relatively far away. It worked great until I did some docking and undocking.

Anyway, managed to get a similar job done by using a washer instead of hinges.

Speaking of pistons, I haven't figured out how to use them right, and the plugin didn't come with stock ships.

**Comment by shrx** [comment]
Also happens with docking washers:
![catastrophic failure](http://i.imgur.com/O4IRpT9.jpg)

so the temporary fix is to not attach them directly to docking ports?
